backend/hasura-actions/src/Scheduler.res

```diff
@@ -144,14 +144,14 @@ let startProcess = () => {
                     let paymentState = payment.paymentState
                     //let paymentAmount = payment.paymentAmount
                     //let paymentTimestamp = payment.paymentTimestamp
-                    if paymentState == "COMPLETE" {
+                    if paymentState == #COMPLETE {
                       Js.log("last payment is COMPLETE")
                       let _ = makePayment(~recipientAddress=recipient, ~paymentData)
                     }
-                    if paymentState == "PENDING" {
+                    if paymentState == #PENDING {
                       Js.log("last payment is PENDING")
                     }
-                    if paymentState == "ERROR" {
+                    if paymentState == #ERROR {
                       Js.log("last payment is ERROR")
                     }
```

Comment:
This should rather be a `switch`

Reply:
(You'll also need to include a `#futuerAddedValue` to this switch, but that is fine - it just future proofs this code so that it will still work even if the schema changes in the future to include another option here.